libmozevent/utils.py

```diff
@@ -175,16 +184,31 @@ class AsyncRedis(object):
     @staticmethod
     async def connect():
         if AsyncRedis.redis.get(None) is None:
+            retry_on_error = [
+                ConnectionError,
+                ExecAbortError,
+                PubSubError,
+                RedisError,
+                ResponseError,
+                TimeoutError,
+                WatchError,
+            ]
+
+            retries = 5
             AsyncRedis.redis.set(
                 await aioredis.from_url(
                     os.environ["REDIS_TLS_URL"],
                     decode_responses=False,
                     ssl_cert_reqs=None,
+                    retry_on_error=retry_on_error,
+                    retry=Retry(NoBackoff(), retries),
```

Comment:
Maybe we should use an exponential backoff, so we give a bit of time for the server to come back up if it's down